### ISSUE TITLE
Incorrect attribute name in pool_scrub_task module 

### ISSUE THREAD

**Issue by bmarinov** [arensb/ansible-truenas#20]
Invoking the module with invalid or not nonexistent pool name will cause the module to run into a syntax error and fail to report the underlying problem:

```
Traceback (most recent call last):
  File "/tmp/ansible_arensb.truenas.pool_scrub_task_payload_qbsery8n/ansible_arensb.truenas.pool_scrub_task_payload.zip/ansible_collections/arensb/truenas/plugins/modules/pool_scrub_task.py", line 186, in main
AttributeError: 'AnsibleModule' object has no attribute 'json_fail'

During handling of the above exception, another exception occurred:
...
```

There is no attribute called `json_fail`, the correct name is `fail_json`. 
- the issue is on this line: https://github.com/arensb/ansible-truenas/blob/main/plugins/modules/pool_scrub_task.py#L186
- same problem here: https://github.com/arensb/ansible-truenas/blob/main/plugins/modules/pool_scrub_task.py#L190

**Comment by arensb** [comment]
Oops!

**Comment by arensb** [comment]
Fixed in [v1.9.5](https://github.com/arensb/ansible-truenas/releases/tag/v1.9.5).
